go.sum

```diff
@@ -79,6 +79,8 @@ github.com/Knetic/govaluate v3.0.1-0.20171022003610-9aa49832a739+incompatible/go
 github.com/MakeNowJust/heredoc v0.0.0-20170808103936-bb23615498cd/go.mod h1:64YHyfSL2R96J44Nlwm39UHepQbyR5q10x7iYa1ks2E=
 github.com/Masterminds/goutils v1.1.0 h1:zukEsf/1JZwCMgHiK3GZftabmxiCw4apj3a28RPBiVg=
```

Comment:
It seems that we still have v1.1.0 in go.sum, while we shouldn't as we have replace v1.1.0 with v1.1.1 in go.mod. I guess that we need to run `go mod tidy` as well to update go.sum.

Reply:
Also, I tried to upgrade `cluster-network-operator` as `goutils` is dependency of `cluster-network-operator`. But upgrading to latest `cluster-network-operator` requires golang-1.18 and causes some code changes due to API changes(eg. apply.ApplyObject). So I think we can simply replace goutils@v1.1.0 with goutils@v1.1.1 currently.
Later we still need to consider upgrading `cluster-network-operator` as it dependencies always lead to some CVE alerts.

Reply:
Hmm I thought that I did.
Antrea PRs are tested for this, maybe it's worthwhile to do it here as well.